x/twap/logic_test.go

```diff
@@ -85,6 +85,66 @@ func (s *TestSuite) TestGetSpotPrices() {
 	}
 }
 
+func (s *TestSuite) TestIntegrationForTwap() {
```

Comment:
Can we add a comment describing what is the context of the integration being tested here?

Reply:
good point, added